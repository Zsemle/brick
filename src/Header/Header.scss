@import '../style/colors.scss';

.header {
  height: 50px;
  background-color: palegoldenrod;
  color: $weird-test-color;
}
