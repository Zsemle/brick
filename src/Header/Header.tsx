import React, { FunctionComponent } from 'react'
import './Header.scss'

const Header:FunctionComponent = ():JSX.Element => {
  return (
    <header className="header">
      <h1>Header</h1>
    </header>
  )
}

export default Header
