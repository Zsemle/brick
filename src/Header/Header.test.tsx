import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';
import Header from './Header';

describe('<Header />', () => {
  it('renders the component', () => {
    const wrapper:ShallowWrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });

});
