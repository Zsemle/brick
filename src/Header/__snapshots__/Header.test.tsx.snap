// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders the component 1`] = `
<header>
  <h1>
    Header
  </h1>
</header>
`;
